apb/files/bundlebindings.crd.yaml

```diff
@@ -0,0 +1,15 @@
+---
+
+apiVersion: apiextensions.k8s.io/v1beta1
+#apiVersion: v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: bundlebindings.automationbroker.io
+spec:
+  group: automationbroker.io
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    plural: bundlebindings
+    singular: bundleebinding
```

Comment:
Is double ee here a typo?

(now exists in couple more yamls: https://github.com/openshift/ansible-service-broker/search?q=bundleebinding
and in OperatorHub.io: https://github.com/operator-framework/community-operators/blob/master/community-operators/automationbroker/bundlebindings.crd.yaml
and some other repos: https://github.com/search?q=bundleebinding&type=Code)